test/test-poll.c

```diff
@@ -637,7 +637,7 @@ TEST_IMPL(poll_bad_fdtype) {
   uv_poll_t poll_handle;
   int fd;
 
-#if defined(_WIN32)
+#if defined(_WIN32) || defined(__MINGW32__)
```

Comment:
Isn't MINGW32 the same as _WIN32?
```suggestion
#if defined(_WIN32)
```

Reply:
`_WIN32` would need to be defined from the command-line of the cmakefile's settings, really. It's not. I'm not confident enough in doing cmakefile changes.

Reply:
Are you sure? I thought this was a built-in definition:
```
$ i686-w64-mingw32-gcc -x c /dev/null -dM -E | grep WIN
#define _WIN32 1
#define __WINT_MAX__ 0xffff
#define __WINT_MIN__ 0
#define __WIN32 1
#define __WINNT 1
#define __WINNT__ 1
#define __WIN32__ 1
#define __SIZEOF_WINT_T__ 2
#define WIN32 1
#define __WINT_TYPE__ short unsigned int
#define __WINT_WIDTH__ 16
#define WINNT 1

$ x86_64-w64-mingw32-gcc -x c /dev/null -dM -E | grep WIN
#define _WIN32 1
#define _WIN64 1
#define __WINT_MAX__ 0xffff
#define __WINT_MIN__ 0
#define __WIN32 1
#define __WIN64 1
#define __WINNT 1
#define __WINNT__ 1
#define __WIN32__ 1
#define __SIZEOF_WINT_T__ 2
#define WIN32 1
#define WIN64 1
#define __WINT_TYPE__ short unsigned int
#define __WINT_WIDTH__ 16
#define WINNT 1
#define __WIN64__ 1
```

(and also because include/uv.h assumes it exists)